### PR TITLE
Update ApiTokenProperty.java

### PR DESCRIPTION
**Pull request by liucc52**
See [#JENKINS-56499](https://issues.jenkins-ci.org/browse/JENKINS-56499)
Proposed changelog entries
Added HTTP session manager page for managing active Jenkins user sessions.
Submitter checklist
  JIRA issue is well described
  Changelog entry appropriate for the audience affected by the change (users or developer, depending on the change). Examples
* Use the Internal: prefix if the change has no user-visible impact (API, test frameworks, etc.)
  Appropriate autotests or explanation to why this change has no tests
  For dependency updates: links to external changelogs and, if possible, full diffs

### CODE REVIEW

**Comment by daniel-beck** [comment]
Please restore the PR template and fill in the placeholders.

File: core/src/main/java/jenkins/security/ApiTokenProperty.java [hunk 1]
```diff
@@ -199,7 +199,7 @@ public boolean matchesPassword(String token) {
             return false;
         }
         
-        tokenStats.updateUsageForId(matchingToken.getUuid());
+      //  tokenStats.updateUsageForId(matchingToken.getUuid());
```
Comment (jsoref) [inline core/src/main/java/jenkins/security/ApiTokenProperty.java]: As a general rule, code shouldn't be commented out, it should either exist or be removed.

That said, I really don't understand your explanation. If you believe something else is doing this already, it'd simplify reviewers work if you provided a stack trace showing that.

Comment (liucc52) [reply core/src/main/java/jenkins/security/ApiTokenProperty.java]: I filed a bug for this [#JENKINS-56499](https://issues.jenkins-ci.org/browse/JENKINS-56499)

**Comment by liucc52** [comment]
i file another pull request for this
